source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -101,19 +101,21 @@ shall be divided into the following groups:
      - Reports situations when the index is out of range. Higher priority
        than ``oneapi::dal::invalid_argument``.
      - No
-   * - ``oneapi::dal::unimplemented_error``
+   * - ``oneapi::dal::unimplemented``
      - Reports errors that arise because an algorithm or a method is not
        implemented.
      - No
-   * - ``oneapi::dal::unavailable_error``
-     - Reports situations when a device or data is not available.
+   * - ``oneapi::dal::unsupported_device``
+     - Reports situations when a device is not supported.
      - No
 
-If an error occurs during function execution after preconditions and invariants
-were checked, it is reported via ``ondeal::runtime_error`` inheritors. oneDAL
-distinguishes errors happened during interaction with OS facilities and errors
-of destination type's range in internal computations, while other errors are
-reported via ``oneapi::dal::internal_error``.
+Errors occured during the execution of oneDAL functionality are represented with
+``oneapi::dal::runtime_error``. Two main groups of errors are distinguished:
```

Comment:
shouldn't we say "shall be distinguished"?

Reply:
fixed